DESCRIPTION

```diff
@@ -58,7 +58,7 @@ Suggests:
     future,
     future.apply,
     knitr,
-    mrgsolve,
+    mrgsolve (>= 1.0.8),
```

Comment:
Just a reminder so that we don't forget to bump this for idata.all.  That came with mrgsolve 1.1.0.